src/IconButton.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import styled from 'styled-components'
+import Icon from './Icon'
+import Button from './Button'
+import PropTypes from 'prop-types'
+
+const TransparentButton = styled(Button)`
+  padding: 0;
+  height: auto;
+  background-color: transparent;
+  color: currentcolor;
+
+  hover: {
+    background-color: transparent;
+  }
+`
+
+const IconButton = props => (
+  <TransparentButton {...props} title={props.title} onClick={props.onClick}>
```

Comment:
Might be better to destructure the Icon-specific props out and pass the rest of the props to the button `({ name, size, color, ...props })` – also setting `title` and `onClick` here is redundant since you pass `{...props}`

Reply:
Cool stuff!